channeldb/db.go

```diff
@@ -315,6 +317,12 @@ func (d *DB) Path() string {
 	return d.dbPath
 }
 
+// SetKeepFailedPayments sets whether failed payments should be deleted.
+// It is only used for testing.
+func (d *DB) SetKeepFailedPayments(keepFailedPayments bool) {
```

Comment:
Can we just use `OptionKeepFailedPayments`?

Reply:
Good point. Done.